docs/contributor-info/errors-with-hooks.md

```diff
@@ -43,3 +45,16 @@ yalc publish
 cd spec/dummy
 yalc link react-on-rails
 ```
+
+#### Multiple `javascript_pag_tag`s
```

Comment:
`javascript_pack_tag`

@samtgarson This is really useful, but this is the wrong place.

@Judahmeek any suggestion for the right place? We have a big doc PR update: https://github.com/shakacode/react_on_rails/pull/1367

and we document the v6 setup in https://github.com/shakacode/react_on_rails_tutorial_with_ssr_and_hmr_fast_refresh

And the real solution is for rails/webpacker to throw an error if the `javascript_pack_tag` is called multiple times.

Reply:
Great! Yeah, the only reason I put it here was that this page comes up in google searches a lot related to react/rails/errors with hooks (which was how I discovered the issue with multiple bundles).

(surely the real _real_ fix is for Webpacker to allow multiple pack tags but just not include duplicate shared packs...?)

Reply:
@Judahmeek how about a troubleshooting document?

Reply:
@justin808 how would such a document differ from this here?

Things to add:

```ruby
# in helper file
module ReactHelper
  def append_javascript_pack(pack_name)
    content_for :view_pack_tags do
      " #{pack_name}"
    end
  end
end
# in partials
<% append_javascript_pack("packs separated by-spaces") %>
# in layout
    <% 
      # Because content_for appends as strings separated by white spaces, pack names are supposed to be space separated too.
      # This way the separation is consistent regardless of their sources.
      pack_tag_args = (yield :view_pack_tags).strip.split(" ")

      pack_tag_args << "oneflare-header" unless skip_header?
      pack_tag_args << "oneflare-footer" unless skip_footer?
      pack_tag_args << "oneflare-user-switcher" if Rails.env.development? || Rails.env.staging?
    %>
    <%= javascript_pack_tag *pack_tag_args unless pack_tag_args.empty? %>
```

Reply:
@Judahmeek the addition above is good.

Might be pretty self-explanatory if this gets merged: https://github.com/rails/webpacker/pull/3083/files

Reply:
@Judahmeek can you close this and a docs page to https://github.com/shakacode/shakapacker/.